ecosystem.core3.config.js

```diff
@@ -51,20 +50,19 @@ module.exports = {
 			autorestart: true,
 			env: {
 				// --- Remember to set the properties below
-				// SERVICE_BROKER: 'redis://localhost:6379/0',
-				// LISK_CORE_WS: 'ws://localhost:8080',
-				// SERVICE_CORE_REDIS: 'redis://localhost:6379/1',
-				// SERVICE_CORE_MYSQL: 'mysql://root@localhost:3306/sdk_v5_betanet',
+				SERVICE_BROKER: 'redis://localhost:6379/0',
+				LISK_CORE_WS: 'ws://localhost:8080',
+				SERVICE_CORE_REDIS: 'redis://localhost:6379/1',
+				SERVICE_CORE_REDIS_VOLATILE: "redis://localhost:6379/8",
+				SERVICE_CORE_MYSQL: 'mysql://root:password@localhost:3306/lisk',
 				LISK_STATIC: 'https://static-data.lisk.com',
-				GEOIP_JSON: 'https://geoip.lisk.io/json',
+				GEOIP_JSON: 'false',
```

Comment:
Why are we setting `GEOIP_JSON` to `false` instead of `''` by default?
```suggestion
				GEOIP_JSON: '',
```